app/test_main.py

```diff
@@ -1,3 +1,25 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_human_age",
+    [
```

Comment:
Please add cases as mentioned in checkilst

- if function resive data out of normal range, such as negative numbers or realy large numbers;
- if the function receives an incorrect type of data it raises the correct exception.